homeassistant/components/bayesian/binary_sensor.py

```diff
@@ -20,18 +26,20 @@
     STATE_UNAVAILABLE,
     STATE_UNKNOWN,
 )
-from homeassistant.core import HomeAssistant, callback
+from homeassistant.core import Event, HomeAssistant, callback
 from homeassistant.exceptions import ConditionError, TemplateError
 from homeassistant.helpers import condition
 import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 from homeassistant.helpers.event import (
     TrackTemplate,
+    TrackTemplateResult,
+    _TrackTemplateResultInfo,
```

Comment:
You shouldn't be importing a private member.

I have a feeling that it shouldn't be private in the first place, so maybe create a preliminary PR to remove the underscore in `homeassistant/helpers/event.py`